pkg/kube/controllers/extendedstatefulset/util.go

```diff
@@ -0,0 +1,44 @@
+package extendedstatefulset
+
+import (
+	"context"
+
+	"k8s.io/api/apps/v1beta2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	crc "sigs.k8s.io/controller-runtime/pkg/client"
+
+	estsv1 "code.cloudfoundry.org/cf-operator/pkg/kube/apis/extendedstatefulset/v1alpha1"
+	"code.cloudfoundry.org/cf-operator/pkg/kube/util/ctxlog"
+)
+
+// listStatefulSets gets all StatefulSets cross version owned by the ExtendedStatefulSet
+func listStatefulSets(ctx context.Context, client crc.Client, exStatefulSet *estsv1.ExtendedStatefulSet) ([]v1beta2.StatefulSet, error) {
```

Comment:
@edwardstudy  I'm wondering if we can put this one and `listSecrets` into the `pkg/kube/apis/extended*/*/types.go` files, or if that would create an import cycle.